README.md

```diff
@@ -58,4 +110,159 @@ are empty or missing, dashboard is effectively disabled.
+
+As a result, `Advanced Module List` option should look like that (provided there are no other advanced modules are 
+enabled for course): 
+![Advanced Module List Image](/docs/images/advanced_module_list.png)
+
+
+### (Optional) Notifications integration
+
+Group Project XBlock v2 sends a number of notifications via edx-notifications app. In order to make those notifications
+appear correctly, make sure `NOTIFICATION_CLICK_LINK_URL_MAPS` Django setting contains the following record:
+
+    'open-edx.xblock.group-project-v2.*': '/courses/{course_id}/group_work?activate_block_id={location}'
+    
+If you're using edx-solutions/edx-platform fork, appropriate value is already set. Otherwise, note that 
+`NOTIFICATION_CLICK_LINK_URL_MAPS` are read from `lms.env.json`, so they should be modified there.    
+
+Example:
+
+```
+NOTIFICATION_CLICK_LINK_URL_MAPS = {
+    'open-edx.studio.announcements.*': '/courses/{course_id}/announcements',
+    'open-edx.lms.leaderboard.*': '/courses/{course_id}/cohort',
+    'open-edx.lms.discussions.*': '/courses/{course_id}/discussion/{commentable_id}/threads/{thread_id}',
+    'open-edx.xblock.group-project.*': '/courses/{course_id}/group_work?seqid={activity_location}',
+    'open-edx.xblock.group-project-v2.*': '/courses/{course_id}/group_work?activate_block_id={location}',
+}
+```  
+
+
+# Configuration
+
+In order to have a working Group Project, one need three components:
+1. Author the XBlock in Studio
+2. Set up environment configuration variables (covered earlier)
+3. Configure Group Project in Apros (3rd party LMS) 
+
+## Authoring
+
+Group Project XBlock v2 relies heavily on "nested XBlocks" feature, provided by edX Platform and XBlocks ecosystem.
+To be precise, Group Project is built from smaller XBlocks, each implementing some smaller subset of Group project 
+functionality: peer grading, submissions upload, team discussions, etc. Most of these features are internal to Group 
+Project XBlock v2 package, but some more advanced features reuse other XBlocks.
+
+Overall, the project structure is the following:
+
+* Group Project XBlock
+    * Group Activity XBlock 1
+        * Stage XBlock 1
+        * Stage XBlock 2
+            * Stage component XBlock
+            * Stage component XBlock
+        * Stage XBlock 3
+    * Group Activity XBlock 2
+        * Stage XBlock 4
+            * Stage Component XBlock
+        * Stage XBlock 5
+    * Project Navigator XBlock
+        * Project Navigation View Xblock
+        * Submissions View XBlock
+        * Resources View XBlock
+        * Private Discussion View XBlock
+    * Discussion XBlock
+    
+In this example Group Project with two activities, Project Navigator and Private Discussions is shown. Activity 1 
+is composed of three stages, Activity 2 is composed of two stages. Some stages have stage components - even smaller
+building blocks, varied by stage type. Project Navigator have most of the Project Navigator available.
```

Comment:
@e-kolpakov "Project Navigator includes most of the child blocks that are available for it"?